idea-plugin/src/main/kotlin/arrow/meta/ide/dsl/resolve/ResolveScopeSyntax.kt

```diff
@@ -0,0 +1,23 @@
+package arrow.meta.ide.dsl.resolve
+
+import arrow.meta.ide.IdeMetaPlugin
+import arrow.meta.phases.ExtensionPhase
+import com.intellij.openapi.extensions.LoadingOrder
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.psi.ResolveScopeEnlarger
+import com.intellij.psi.search.SearchScope
+
+interface ResolveScopeSyntax {
+    fun IdeMetaPlugin.resolveScope(
+        getAdditionalResolveScope: (file: VirtualFile, project: Project) -> SearchScope
```

Comment:
```suggestion
        additionalResolveScope: (file: VirtualFile, project: Project?) -> SearchScope?
```
Nit's in the type signature

Reply:
thanks!

Reply:
I updated the suggestion from above. But that means to change the signature in the overridden function, too.